config/application.rb

```diff
@@ -40,5 +40,11 @@ class Application < Rails::Application
     ## currently only using a single instance (the Publishing "shared" Redis). If we ever need to
     ## use multiple Redis instances, this is the only place that needs updating.
     config.redlock_redis_instances = [config.redis_url]
+
+    # Feature flags
+    def self.feature_flag(name)
+      ActiveModel::Type::Boolean.new.cast(ENV.fetch(name, false))
```

Comment:
Nice idea to use the Boolean type cast in the great is an env var a boolean or not debate (wish there was a handy method on ENV)

You might want to use this as an opportunity to flip the environment variable so that rather than us having a `GOVUK_SEARCH_DISABLE_AUTOCOMPLETE=1` we instead have `GOVUK_SEARCH_ENABLE_AUTOCOMPLETE=false` - as my use of a disable env var proved rather contentious.

Reply:
Yeah, I was torn on that too and also prefer "positive" naming.

My rationale here was that by using the same env var we'd make it easier to do the next steps of the migration, but having slept on it there's really no reason that makes anything any easier so will change it to avoid having to change it again in the future!